node/src/mpc_client.rs

```diff
@@ -272,6 +272,13 @@ impl MpcClient {
                     .process_channel(channel)
                     .await?
             }
+            MpcTaskId::EddsaTaskId(_) => {
+                unreachable!()
+                // self.eddsa_signature_provider
+                //     .clone()
+                //     .process_channel(channel)
+                //     .await?
```

Comment:
Why is this not implemented?

Reply:
I played with this a bit. We're lacking support for storing keyshares for eddsa and hooking up the keygen and resharing with the keyshare (this needs the eddsa KeygenOutput to be serializable). On the keygen/resharing side, we're missing a DomainRegistry parameter, otherwise it doesn't know what kind of key to generate (it has the DomainId but not the DomainConfig which contains the SignatureScheme). So yeah there's still some hooking up to do.